iroh-net/src/netcheck/reportgen.rs

```diff
@@ -170,6 +170,12 @@ struct Actor {
     last_report: Option<Arc<Report>>,
     /// The portmapper client, if there is one.
     port_mapper: Option<portmapper::Client>,
+    /// Whether the actor should try to reach things other than localhost.
+    ///
+    /// This is set to true in tests to avoid probing the local LAN's router etc.
+    ///
+    /// TODO: currently this only skips the portmapper, but still does STUN, HTTP and ICMP
+    /// probes.  Furthermore none of our tests actually use this feature.
```

Comment:
honestly, just delete it, we can bring it back if we ever need to

Reply:
yeah, fair enough.  i'll do that in a separate PR though.